CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Release 0.4.0
```

Comment:
This hasn't been released yet - perhaps this can be part of the release action to generate the file?

Reply:
Oh thanks for catching it! Yeah, if that's the case, let me take this file out. Sounds good, I can let the `cli.py` touch this file before trying to open, and Action can just use the `cli.py`'s `release` command.